.github/workflows/tests.yml

```diff
@@ -85,8 +85,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer require dunglas/symfony-lock:v${{ matrix.symfony }} --no-interaction --no-update
-          composer update --prefer-dist --no-interaction --prefer-stable --prefer-lowest --no-progress
+          SYMFONY_REQUIRE=${{ matrix.symfony }} composer update --prefer-dist --no-interaction --prefer-stable --prefer-lowest --no-progress
```

Comment:
afaik we need a line before this one like

            composer require --no-update symfony/flex && composer config extra.symfony.require ${{ matrix.symfony}}

Reply:
Seems to work at least locally. I've committed the changes.